ai/core/src/unitycatalog/ai/core/databricks.py

```diff
@@ -588,103 +567,7 @@ def _execute_uc_function(
         self, function_info: "FunctionInfo", parameters: Dict[str, Any], **kwargs: Any
     ) -> Any:
         check_function_info(function_info)
-        if self.warehouse_id:
-            return self._execute_uc_functions_with_warehouse(function_info, parameters)
-        else:
-            return self._execute_uc_functions_with_serverless(function_info, parameters)
-
-    @retry_on_session_expiration
-    def _execute_uc_functions_with_warehouse(
-        self, function_info: "FunctionInfo", parameters: Dict[str, Any]
-    ) -> FunctionExecutionResult:
-        from databricks.sdk.service.sql import StatementState
-
-        _logger.info("Executing function using client warehouse_id.")
-
-        parametrized_statement = get_execute_function_sql_stmt(function_info, parameters)
```

Comment:
Seems like get_execute_function_sql_stmt can be dropped as well

Reply:
Good catch!